### DISCUSSION TITLE
https://github.com/limboemu/limbo

### DISCUSSION THREAD

**Discussion by SDRausty** [termux/termux-packages#7756] [Q&A] [answered]
>Limbo is a QEMU-based emulator for Android. It currently supports x86, ARM, PowerPC, and Sparc emulation for Intel x86 and ARM android devices. See wiki https://virtualmachinery.weebly.com for APK download and Instructions.
Languages:
Java 56.0% 
C 39.6% 
Makefile 3.4% 
C++ 1.0%

ASM?
ASM translator?

Issues: https://github.com/termux/termux-packages/issues?q=limbo

> [have tried even limbo emulator, same issue.](https://github.com/termux/termux-packages/issues/6651)

How can [`limbo`](https://github.com/limboemu/limbo ) be used with https://github.com/termux/proot?

**Comment by ghost** [comment, marked as answer]
Limbo is a QEMU (system mode) port to Android OS.

Not perfect though. The available `qemu-*` packages in Termux main and x11 repositories work much better and far more flexible.

> How can limbo be used with https://github.com/termux/proot?

PRoot can be used only with QEMU user-mode emulators.

***

Discussion is locked as off-topic (third-party project that has Termux-packaged analog or exact copy).
